arch/x86/core/Kconfig.intel64

```diff
@@ -32,8 +32,10 @@ config X86_EXCEPTION_STACK_SIZE
 config X86_EXCEPTION_STACK_TRACE
 	bool
 	default y
+	select DEBUG_INFO
+	select NO_OPTIMIZATIONS
```

Comment:
@dcpleung I also added a commit to select `NO_OPTIMIZATIONS` here instead of depending on it

Reply:
That was a bad idea, removed